integration-tests/http/__tests__/tax-region/admin/tax-region.spec.ts

```diff
@@ -20,12 +20,21 @@ medusaIntegrationTestRunner({
         let taxRegion
 
         beforeEach(async () => {
+          const parentRegion = await api.post(
```

Comment:
suggestion: should we also add a test ensuring that we can create a province region without a provider? It seems to be missing

Reply:
this is tested implcitley in `beforeEach` where we create a parent tax region and a province region, but I added an explicit case as well